server/src/router.js

```diff
@@ -19,6 +20,7 @@ router.post('/promo', decodeToken, createPromoCode);
 router.get('/meals/category/:id', getMealsByCategory);
 router.get('/meals/restaurant/:id', getMealsByRestaurant);
 router.get('/meals/:mealid', getMealById);
+router.get('/restaurants/search/:name', searchRestaurant);
```

Comment:
how about GET `/restaurants?search=<query>

Reply:
wooooow
greate idea 
I have to edit all the code now :)

Reply:
the route will be `GET /restaurants`

Reply:
querystrings can be accessed from req.query

Reply:
yea yea I understand